consensus/poet/core/tests/test_consensus/test_poet_block_publisher.py

```diff
@@ -309,11 +309,11 @@ def test_k_policy(
 
         # create a mock_consensus_state that returns a mock with
         # the following settings:
-        mock_state = mock.Mock()
-        mock_state.validator_signup_was_committed_too_late.return_value = False
-        mock_state.validator_has_claimed_block_limit.return_value = True
-        mock_state.validator_is_claiming_too_early.return_value = False
-        mock_state.validator_is_claiming_too_frequently.return_value = False
+        mock_state = MockConsensusState().create_mock_consensus_state(
+            committed_too_late=False,
+            claimed_block_limit=True,
+            claiming_too_early=False,
+            claiming_too_frequently=False)
```

Comment:
For example, here you would do:
`mock_state = MockConsensusState.create_mock_consensus_state(claimed_block_limit=True)`